content/articles/setting-up-hotspot-on-kali-linux/index.md

```diff
@@ -0,0 +1,149 @@
+```bash
+sudo gedit /etc/dnsmasq.conf
+```
+After running the command, write the code below under **`hotspot.sh`** for configuration:
+```
+# Only bind it to one interface
+bind-interfaces
+# Select the interface to use for binding
+interface=wlan0
+# Select a scope of IP addresses to be used in DHCP leasing
+
+dhcp-range=192.168.1.1,192.168.1.9
+```
+The preceding step in this stage is performing hostapd configuration. 
+In order to perform hostapd configuration, type in the following command in the terminal.
+```bash
+sudo gedit/etc/hostapd.conf
+```
+To properly configure **hostapd**, add the the following code to our configuration file:
+```
+# Set interface
+interface=wlan0
+# Set driver to
+driver=nl80211
+# Set your desired ssid(Wi-Fi name)
+ssid=Wi-FiConnection
+# Set the access point hardware mode to 802.11g
+hw_mode=g
+# Select WIFI channel
+ channel=6
+# Ensure to enable only WPA2 
+wpa_passphrase=Wi-Fi connection
+```
+You can change  your`SSID` name and password with your desired choice. The above configuration will create a hotspot called *Wi-Fi connection* with  a password.
+>N/B:  You can create the configuration file anywhere you want  named  **`hotspot.sh`** using an editor of your choice.
+
+After completing the configuration, we shall stop the running services by  typing in the commands below:
+```bash
+sudo service dnsmasq stop
+```
+```bash
+sudo service hostapd stop
+```
+For a wired connection, you will be required to change **ppp0** to **eth0** or any other number.
+
+### Using GUI
+
+In this part, we will be using the graphical interface to do the configurations.
+- Go to advanced network settings, click on the gear icon to open it up. Your screen should look like this: 
+
+![Settings](/engineering-education/setting-up-hotspot-on-kali-linux/settings.png)
+To proceed with our configuration, follow the steps below:
+**Step I**: In the window, you click the “+”  button and another window pops up.
+In the new window here we are going to do the configuration of your hotspot.
+![Connection](/engineering-education/setting-up-hotspot-on-kali-linux/connection.png)
+**Step II**: Under the network type select “Wi-Fi”.
+Ensure your configuration resembles the second screenshot  below
+
+![Security](/engineering-education/setting-up-hotspot-on-kali-linux/security.png) 
+**Step III**: To set a password, click on **“Wi-Fi security ”** and select **“ WPA & WPA2 Personal”**
+>N/B: Security being an essential  feature, this type of encryption protection is designed for homes and small office networks and does not need authentication server, also  known as WPA-PSK(pre-shared key). WPA Personal is available on all 3 WPA versions that are; WPA, WPA2, and WPA3) 
+
+and below the input box give an 8 or more memorable digit password of your choice “********” and click on save.
+
+### Conclusion
+
+Finally, you've done it! Close all the opened windows and enjoy your newly created hotspot, share it amongst computers, mobile phones, etc.
```

Comment:
A couple of corrections here:
1. You cannot start a sentence with "and"
2. This sentence in your conclusion should be cut and pasted before the conclusion. It summarizes what your tutorial was about.
3. Once you've removed the sentence and pasted as advised, you can now write an elaborate conclusion.